collections/_posts/2024/2024-12-18-meshery-v0.8.0-release-announcement.md

```diff
@@ -0,0 +1,151 @@
+
+* **Enhanced User Experience:** Enjoy a more intuitive and visually appealing interface.
+* **New Components:** Leverage a wide range of new components from the Sistent design system.
+
+### CLI: Increased Coverage and Functionality
+
+The Meshery CLI boasts increased code coverage and a full deprecation of Apps and migration to Designs.
+
+* **Enhanced Model Support:** Dedicated commands for managing models, relationships, components, connections, credentials, and policies.
+
+### Documentation, Learning, and the Playground
+
+* **Playground Learning Paths:**  Interactive, cloud-native tutorials within Meshery Playground offer hands-on learning experiences.
+- Remote Provider: Deployment using Meshery (as a Meshery Design)
+- Remote Provider: Multi-master Postgres
+
+## Docs
+
+* [Playground] Learning Paths: Cloud Native tutorials with Meshery Playground
+  * Creating three tutorials per category, allowing learners to perform hands-on lab interactively in the live cluster environment in Meshery Playground.
+* Automated publication of End-to-End test results
+
+### Build, Test, Release: Improved Processes
+
+* **End-to-End Testing:**  Increased focus on end-to-end testing with a goal of 50% coverage.
+* **Playwright Adoption:**  Transitioning from Cypress to Playwright for more robust testing.
+* **Automated Test Results:** Stay informed with automatically published end-to-end test results.
+
+### Join the Meshery Community!
+
+We invite you to explore Meshery v0.8.0 and experience the future of cloud native management.  Join us on [Slack](https://slack.meshery.io/) to connect with the community, share your feedback, and contribute to the project.
+
+New Maintainer: [Sangram Rath](https://layer5.io/community/sangram-rath)
+
+**Get Involved!**
+
+We welcome your contributions to Meshery! Please join our <a href="/calendar">weekly newcomers meeting</a> for a guided tour of how to get involved.
+
+ _If these topics excite you, come and say "Hi" in the community [Slack](https://slack.meshery.io/) and you are sure to be warmly welcomed. 😀_
+
+**Thank You!**
+
+A big thank you to all of the contributors who made this release possible! We are grateful for your continued support of the Meshery project.
+
+## Looking Ahead and onto v0.9
+
+While v0.8.0 delivers significant advancements, we're already working on v0.9.0! 
+
+<!-- Checkout [Meshery’s roadmap](https://github.com/meshery/meshery/blob/master/ROADMAP.md) to peek into what’s yet to come. -->
+
+We are already hard at work on [v0.9.0 Roadmap](https://discuss.layer5.io/t/meshery-v0-9-roadmap/6296)! See [v0.9.0 Milestone](https://github.com/meshery/meshery/milestone/5) in `meshery/meshery` for more details and the [Meshery Architecture deck](https://docs.google.com/presentation/d/1SQMfyu5shjpGKlYONdVzOtd7UYTgLWBcgUvHMLCZ2tY/edit#) for additional visual designs and functional specifications. Stay tuned for more updates.
```

Comment:
The milestone link should be https://github.com/meshery/meshery/milestone/6, isn't it?

Reply:
Oh, my goodness. Yes, thank you. Updated.